### PR TITLE
Make link a link on the getting started

### PR DESCRIPTION
**Pull request by Davidnet**
# What does this PR do?

Make a link a clickable link

### CODE REVIEW

**Review by cgarciae — APPROVED** [review]
Thanks @Davidnet! LGTM.

File: docs/getting_started.md [hunk 1]
```diff
@@ -403,4 +403,4 @@ Congrats! You made it to the end of the annotated MNIST example. You can revisit
 the same example, but structured differently as a couple of Python modules, test
 modules, config files, another Colab, and documentation in Flax's Git repo:
 
-https://github.com/google/flax/tree/main/examples/mnist
+[https://github.com/google/flax/tree/main/examples/mnist](https://flax.readthedocs.io/en/latest/getting_started.html)
```
Comment (levskaya) [inline docs/getting_started.md:406]: could we link to the github page for mnist directly here? (ie the same url)

**Review by levskaya — CHANGES_REQUESTED** [review]

File: docs/getting_started.md (hunk 1 above)
Comment (Davidnet) [reply docs/getting_started.md]: Perfect I think I have added that.

**Review by levskaya — APPROVED** [review]

**Comment by levskaya** [comment]
Side note:  we'll take this as-is since it's small, but in the future when submitting PRs to flax or any other codebase using github/gitlab/etc please know that it's helpful to "squash" all of your commits together into a _single_ commit locally and/or doing a force push to keep the PR 1:1 with a _single_ commit.   This keeps the repository history cleaner (easier to use `git blame`, etc.) and helps maintainers keep track of changes.

**Comment by Davidnet** [comment]
@levskaya Feedback well received, thanks.

**Pull request merged**